src/System/Integrate/Console/HelpCommand.php

```diff
@@ -263,11 +261,9 @@ public function commandHelp(): int
      */
     private function commandMaps()
     {
-        $commandmap =  [];
-        foreach ($this->commands as $command) {
-            $commandmap[] = new CommandMap($command);
-        }
-
-        return $commandmap;
+        return array_map(
+            static fn ($command): CommandMap => new CommandMap($command),
+            Application::getIntance()[ConfigRepository::class]->get('commands', [])
+        );
```

Comment:
intead using `Application::getIntance()` better using application parameter

Reply:
for now it is oke, the problem is parent `Command::main()` doesn't required any parameters. if it change will throw error due child not same with parent class.

The solutions is make pr to remove `Command::main()` from Command. so child can use costume parameter.

or

dont use main methods to register this signature.